source/expressions.tex

```diff
@@ -1589,7 +1589,7 @@
 auto x1 = [](int i){ return i; };     // OK: return type is \tcode{int}
 auto x2 = []{ return { 1, 2 }; };     // error: deducing return type from \grammarterm{braced-init-list}
 int j;
-auto x3 = []()->auto&& { return j; }; // OK: return type is \tcode{int\&}
+auto x3 = [&j]()->auto&& { return j; }; // OK: return type is \tcode{int\&}
```

Comment:
This seems to be moving comments around; can you please adjust everything according to our usual comment placement guidance?

Reply:
I'm sorry. Just fixed the indent.

Reply:
Thanks - [see wiki](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#code-blocks), that preferred comment columns are 20, 32, 40, 48, etc. Can you snap to one of those?